Gems/PhysX/Code/Editor/EditorJointConfiguration.h

```diff
@@ -71,6 +71,24 @@ namespace PhysX
         float m_limitNegative = -45.0f;
     };
 
+    /// Pair (angles) limits for joints.
+    class EditorJointMotorConfig
```

Comment:
nit: I believe most classes use the full `Configuration` as opposed to the abbreviated `Config`, might be nice to change this just to be consistent (if there's already a mix elsewhere feel free to ignore this)

Reply:
In this file there is `EditorJointLimitConfig` and `EditorJointLimitPairConfig`, so having Config here is fine to be consistent.